api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -148,14 +144,19 @@ Random random() {
 
   @Captor private ArgumentCaptor<Map<String, String>> mapCaptor;
 
-  @Autowired ClusterAuditor clusterAuditor;
-  @Autowired LeonardoNotebooksClient notebookService;
-  @Autowired FireCloudService fireCloudService;
+  @MockBean AdminActionHistoryDao adminActionHistoryDao;
+  @MockBean ClusterAuditor clusterAuditor;
+  @MockBean ComplianceService complianceService;
+  @MockBean DirectoryService directoryService;
+  @MockBean FireCloudService fireCloudService;
+  @MockBean LeonardoNotebooksClient notebookService;
```

Comment:
Is this name intentional? Looks like there's a separate `org.pmiops.workbench.notebooks.NotebooksService`.